### ISSUE TITLE
No option to save model; saving in main directory

### ISSUE THREAD

**Issue by ajaysub110** [SforAiDl/JigglypuffRL#58]
- There seems to be no explicit option to save a model. 
- Instead of saving the model checkpoint files in the main directory, it would be better to do so inside a new directory named `checkpoints` or so

**Comment by Zeus3101** [comment]
I'll try to fix this

**Comment by Zeus3101** [comment]
Should I use argparse to create a separate option when running the code itself?

**Comment by ajaysub110** [comment]
No need to add argparse now in my opinion. It can be done in the same way we're handling tensorboard. The algo class can take in 2 arguments: `save_model=None` (str) and `save_interval`. (int)
if `save_model` is `None`, then no need to save, else, save it in the location specified by it.
`save_interval` just takes in a number saying how many iterations/episodes between saves, as has already been done. 
Does this make sense?

**Comment by Zeus3101** [comment]
Yeah, will do.

**Comment by Sharad24** [comment]
Yes, we'd like to avoid argparse for now, especially at the core of the library since it does not make sense. Maybe we can have it examples.

On another note, we should shift the following code to examples as well:
```
if __name__ == "__main__":
    algo = Algo()
    algo.learn()
    etc
```

**Comment by Sharad24** [comment]
Closed via #63
